doc/manual/98-licenses.md

```diff
@@ -38,3 +38,36 @@ OF SUCH DAMAGE.
 ```
 
 [BSD-3-Clause][license]
+
+
+## libart
+Our adaptive radix tree (ART) implementation is based on
+ [The Adaptive Radix Tree: ARTful Indexing for Main-Memory Databases][ART_paper]
+ and [libart][libart] which has a [3-BSD license][license] as
+ 
+```
+Copyright (c) 2012, Armon Dadgar
+All rights reserved.
+
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are met:
```

Comment:
Format this a little such that it looks nicer in the manuals

Reply:
Adjusted the length of each line according to ours.